client/mmadness/src/login/login.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import './login.css';
+import logo from './../img/MM-logo.png';
+
+export default class Login extends Component {
+  state = {
+    errorStyle:''
+  }
+  
+  userLogged(event){
+    event.preventDefault();
+    const username = document.getElementById('userIs').value.toUpperCase();
+    const roomCode = document.getElementById('roomCodeIs').value.toUpperCase();
```

Comment:
Accessing the DOM is slow and imperative, better go with [Controlled Components](https://reactjs.org/docs/forms.html#controlled-components)